doc/nrf/ug_thread_tools.rst

```diff
@@ -53,7 +53,16 @@ The interaction is possible using commands proper to wpanctl, a module installed
 Installing wpantund
 ===================
 
-For installation and initial configuration, see `wpantund Installation Guide`_.
+To ensure that the interaction with the samples works as expected, you should install the version of wpantund that has been used for testing the |NCS|.
+
+See the `wpantund Installation Guide`_ for general installation instructions.
+To install the correct version, replace the ``git checkout full/latest-release`` command with the following command:
+
+.. parsed-literal::
+
+   git checkout 87c90eedce0c75cb68a1cbc34ff36223400862f1
+
+Note that when installing on macOS, you must follow the instructions for the manual installation and replace the above command to ensure that the correct version is installed.
```

Comment:
It supports macOS and Linux, I think we should mention both or none of them.

Reply:
For Linux, the general instructions above apply. For macOS, the problem is that there are two ways of installing, and you can only ensure the correct version with the manual installation.
Not sure how to make that clearer - maybe in line 58: "See the `wpantund Installation Guide`_ for general installation instructions for Linux and macOS."
Would that help?

Reply:
You are right I missed that.
I agree that installing with brew should be avoided and the manual installation is the way to go.

No changes required :)